Makefile

```diff
@@ -11,7 +11,7 @@ LDFLAGS += -X "github.com/pingcap/pd/server.PDGitHash=$(shell git rev-parse HEAD
 # Ignore following files's coverage.
 #
 # See more: https://godoc.org/path/filepath#Match
-COVERIGNORE := "cmd/pd-server/main.go,server/api/bindata_assetfs.go"
+COVERIGNORE := "cmd/*/*,pdctl/*,pdctl/*/*,server/api/bindata_assetfs.go"
```

Comment:
No, they don't.

`pdctl/*` for `pdctl/ctl.go`
`pdctl/*/*` for `pdctl/command/*.go`

See more: 
 + https://godoc.org/path/filepath#Match
 + https://golang.org/src/path/filepath/match_test.go?#L24